architecture/spmc.rst

```diff
@@ -283,6 +283,223 @@ needs access to the rxtx buffer.
 A separate ``struct rxtx_buf`` is defined for the Normal World, which gives
 access to the Normal World buffers.
 
+FF-A compliance
+===============
+
+.. |ffa_fs| unicode:: U+02714
+.. |ffa_ps| unicode:: U+0237B
```

Comment:
This doesn't render in my browser.

Reply:
I went with this unicode based solution because it doesn't introduce any new build dependencies but I wasn't aware that it has compatibility issues.
If it's acceptable to introduce the use of a new sphinx addon, [sphinx-panels' inline icons](https://sphinx-panels.readthedocs.io/en/latest/#inline-icons) can solve the issue. Github octicons use an SVG format for rendering the icons so it doesn't depend on the system fonts. It looks like this:
![image](https://user-images.githubusercontent.com/50944609/221180693-777a9f28-9022-4d1b-aa67-7bab94090e0b.png)

Reply:
> If it's acceptable to introduce the use of a new sphinx addon, [sphinx-panels' inline icons](https://sphinx-panels.readthedocs.io/en/latest/#inline-icons) can solve the issue.

I can add that @imre-kis-arm , let's keep this PR open until I've given it a try to configure that.